browser/caliban.py

```diff
@@ -972,8 +986,8 @@ def action_watershed(self, label, frame, x1_location, y1_location, x2_location,
         new_label = max(self.tracks) + 1
 
         # Locally store the frames to work on
-        img_raw = self.raw[frame]
-        img_ann = self.tracked[frame]
+        img_raw = self.raw[frame,:,:,0]
+        img_ann = self.tracked[frame,:,:,0]
```

Comment:
is this future-proofed against RGB? Why call out the channel specifically?

Reply:
Calling out channel 0 is for trk files specifically; I don't think we support multiple channels or features for trk; it was just implicit in this part of the code before. Indexing into channel 0 specifically keeps the image 2D, which helps do the dilation consistently. Otherwise, some of the dilation/np.where code would need to be reworked to use 3D arrays. Having it this way makes it consistent with npz watershed which should make a future refactor easier.